bindings_ffi/src/mls.rs

```diff
@@ -335,24 +337,25 @@ pub struct FfiListMessagesOptions {
 impl FfiGroup {
     pub async fn send(&self, content_bytes: Vec<u8>) -> Result<Vec<u8>, GenericError> {
         let group = MlsGroup::new(
```

Comment:
With these changes, we can't just store an `Arc<MlsGroup>` on the `FfiGroup` and remove this silliness around having to re-create the group on each method call?

That was purely there because of lifetime issues with Uniffi

Reply:
Good call, that's definitely worth doing now that we don't have to deal with the lifetimes

Reply:
Can be a separate PR